chrome/Dockerfile

```
@@ -0,0 +1,39 @@
+FROM ubuntu:bionic
+
+ARG VERSION
+ARG DEBIAN_FRONTEND=noninteractive
+ENV RD_PORT=9222
+
+RUN apt-get update \
+    # https://github.com/phusion/baseimage-docker/issues/319
+    && apt-get install --yes apt-utils 2>&1 | grep -v "debconf: delaying package configuration, since apt-utils is not installed" \
+    && apt-get install -y wget gnupg \
+    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install --no-install-recommends -y \
+      google-chrome-stable \
+      fonts-ipafont-gothic \
+      fonts-wqy-zenhei \
+      fonts-thai-tlwg \
+      fonts-khmeros \
+      fonts-kacst \
+      fonts-freefont-ttf \
+      libxss1 \
+      htop \
+      net-tools \
+      dumb-init \
+      curl \
+      socat \
+    && rm -rf /var/lib/apt/lists/* \
+    && groupadd chrome \
+    && useradd --create-home --gid chrome chrome \
+    && chown --recursive chrome:chrome /home/chrome/
```

Extract them as separate instructions as follows:

```suggestion
    && rm -rf /var/lib/apt/lists/*
    
RUN groupadd chrome
RUN useradd --create-home --gid chrome chrome
RUN chown --recursive chrome:chrome /home/chrome/
```

---

It was split.